plugins/basic/python/tk_maya_basic/plugin_engine.py

```diff
@@ -83,12 +83,14 @@ def bootstrap(sg_user, progress_callback, completed_callback, failed_callback):
 
     logger.debug("Will launch the engine with entity: %s" % entity)
 
-    # Remove the custom logging handler now that the engine will take over logging.
-    sgtk.LogManager().root_logger.removeHandler(plugin_logging_handler)
-
     # Install the bootstrap progress reporting callback.
     toolkit_mgr.progress_callback = progress_callback
 
+    # install a callback that turns off logging just before the
+    # engine starts up its own logging
+    callback = lambda ctx, log_handler=plugin_logging_handler: _pre_engine_start_callback(ctx, log_handler)
```

Comment:
i think it's doing that already - the issue is that with this setup, you end up with double logging when the engine kicks in.

This will be an issue with all plugins - maybe we should create a separate ticket to investigate making this workflow easier? like you say, it should be as simple as the plugin turning on logging and then that's it. No fiddling with turning off and on stuff etc.